kpi/deployment_backends/kc_access/shadow_models.py

```diff
@@ -317,10 +317,13 @@ def sync(cls, user):
         Creates rows when the user is created so that the Admin UI doesn't freak
         out because it's looking for a row that doesn't exist
         """
-        today = datetime.today()
-        first = today.replace(day=1, hour=0, minute=0, second=0, microsecond=0)
+        today = date.today()
+        first = today.replace(day=1)
 
-        cls.objects.get_or_create(user_id=user.pk, timestamp=first)
+        queryset = cls.objects.filter(user_id=user.pk, timestamp=first)
+        if not queryset.exists():
+            # Todo: Handle race conditions
```

Comment:
As per our discussion, @JacquelineMorrissette will open a new issue to handle those.